utils/file-download.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const { existsSync } = require('fs');
+const path = require('path');
+const download = exports;
+
+download.responseFile = async (basePath, fileName, res) => {
+  const fullFileName = path.join(basePath, fileName);
+  const baseDir = path.resolve(__dirname, '..');
+  const directory = path.join(baseDir, fullFileName);
+
+  if (existsSync(directory)) {
+    const filename = path.basename(fullFileName);
+
+    res.setHeader('Content-Disposition', `attachment; filename=${filename}`);
+    res.setHeader('Content-Transfer-Encoding', 'binary');
+    res.setHeader('Content-Type', 'application/octet-stream');
+
+    res.sendFile(directory);
```

Comment:
I think using res.download() will be more concise way to handle file downloads, and provided callback to handles any errors

```suggestion
    res.download(directory, fileName, (err) => {
      if (err) {
        const errorMsg = `Error occurred on file download: ${err.message},
          Cause: ${err.response.status} ${err.response.statusText}`;
        logger.error(errorMsg);
        throw new Error(errorMsg);
      }
    });
```